openmetadata-ui/src/main/resources/ui/src/utils/EntitySummaryPanelUtils.tsx

```diff
@@ -65,12 +65,17 @@ export const getFormattedEntityData = (
         name: chart.name,
         title: (
           <Link target="_blank" to={{ pathname: chart.chartUrl }}>
-            <Space className="m-b-xs">
-              <Text className="entity-title text-primary font-medium">
+            <div className="d-flex items-center m-b-xs">
+              <Text className="entity-title text-primary font-medium m-r-xss">
                 {getTitleName(chart)}
               </Text>
-              <SVGIcons alt="external-link" icon="external-link" width="12px" />
-            </Space>
+              <SVGIcons
+                alt="external-link"
+                height="14px"
+                icon="external-link"
+                width="14px"
+              />
+            </div>
           </Link>
```

Comment:
to reduce dom complexity we can try this 
```
<Link
            style={{ display: 'inline-flex', gap: '8px' }}
            className="tw-items-center m-b-xs"
            target="_blank"
            to={{ pathname: chart.chartUrl }}>
            <Text className="entity-title text-primary font-medium">
              {getTitleName(chart)}
            </Text>
            <SVGIcons alt="external-link" icon="external-link" width="12px" />
          </Link>
```

Reply:
Thanks @ShaileshParmar11 will do it in next iteration